client/extensions/woocommerce/state/wc-api/error-reducer.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_API_ERROR,
+	WOOCOMMERCE_API_ERROR_CLEAR,
+} from '../action-types';
+
+const debug = debugFactory( 'woocommerce:wc-api:error' );
+
+export default {
+	[ WOOCOMMERCE_API_ERROR ]: apiError,
+	[ WOOCOMMERCE_API_ERROR_CLEAR ]: clearApiError,
+};
+
+function apiError( siteState, action ) {
+	const { data, originalAction, time } = action.payload;
+	const newError = { data, originalAction, time };
```

Comment:
I don't see how time is evaluated by the reducer - seems like the last action will always win regardless of the value of `time`